model/vm_host.rb

```diff
@@ -13,6 +13,7 @@ class VmHost < Sequel::Model
   one_to_many :spdk_installations, key: :vm_host_id
   one_to_many :storage_devices, key: :vm_host_id
   one_to_many :pci_devices, key: :vm_host_id
+  one_to_many :boot_images, key: :vm_host_id
 
   plugin :association_dependencies, assigned_host_addresses: :destroy, assigned_subnets: :destroy, hetzner_host: :destroy, spdk_installations: :destroy, storage_devices: :destroy
```

Comment:
Can you add `boot_images` here? They should be deleted when the vm host deleted.
https://github.com/ubicloud/ubicloud/blob/f5cea35e33df427dbe3402b19b31dba7039ade4d/model/vm_host.rb#L17